### PR TITLE
Make the anitya_cron multithreaded and drop support for the progress-bar

### PR DESCRIPTION
**Pull request by pypingou**
With this version, using 10 threads running the cron drops from 90minutes
to less than 10 minutes.

### CODE REVIEW

File: files/anitya_cron.py [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
-import sys
 
-PBAR = True
-try:
-    from progressbar import Bar, ETA, Percentage, ProgressBar, RotatingMarker
-except ImportError:
-    PBAR = False
+def update_project(project):
+    """ Check for updates on the specified project. """
+    print(project.name)
```
Comment (ralphbean) [inline files/anitya_cron.py]: You always run as python3?  or is there a ``from __future__ import print_function`` here that I'm missing.

File: files/anitya_cron.py [hunk 2]
```diff
@@ -41,28 +51,8 @@ def main(debug):
...
-        try:
-            anitya.check_release(project, session)
-        except anitya.lib.exceptions.AnityaException as err:
-            pass
-        cnt += 1
-        if PBAR:
-            pbar.update(cnt)
-    if PBAR:
-        pbar.finish()
+    p = multiprocessing.Pool(anitya.app.APP.config.get('CRON_POOL', 10))
```
Comment (ralphbean) [inline files/anitya_cron.py]: Can you check to see if there is a performance difference between ``multiprocessing.Pool`` and ``multiprocessing.pool.ThreadPool``?

**Comment by ralphbean** [comment]
:+1: as-is, with two non-blocking questions above.

File: files/anitya_cron.py (hunk 1 above)
Comment (pypingou) [inline files/anitya_cron.py]: it seems that ``print()`` works fine in python 2.7 without the import of ``__future__`` (tested on pkgdb02 with python 2.7.5).

That being said, it should be a call to the logger anyway :)

File: files/anitya_cron.py (hunk 2 above)
Comment (pypingou) [inline files/anitya_cron.py]: With ``multiprocessing.Poll``:
``
PYTHONPATH=. python files/anitya_cron.py  163.72s user 15.07s system 27% cpu 10:49.97 total 
``
With ``multiprocessing.pool.ThreadPool``:
``
PYTHONPATH=. python files/anitya_cron.py  163.99s user 22.22s system 22% cpu 14:05.40 total
``

**Pull request merged**